src/main/java/Parser.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Class to parse the input by the user into a command.
+ */
+public class Parser {
+
+    /**
+     * Takes in the full string and processes the string into a command.
+     * @param fullCommand input by the user.
+     * @return Command to be executed later.
+     * @throws DukeException if the command is unknown.
+     */
+    public static Command parse(String fullCommand) throws DukeException {
```

Comment:
Interesting method using Switch. Instead of `StringBuilder`, could try using `substring`, `split`, etc to simplify the parsing of the command strings.